### DISCUSSION TITLE
Difference between categories and tags

### DISCUSSION THREAD

**Discussion by Sleepful** [firefly-iii/firefly-iii#9117] [Firefly III]
It seems both serve the exact same purpose, are there any specific situations where I should use one over the other?

**Comment by JC5** [comment]
Not really, but the default financial report will also report on categories, and not on tags.

**Reply by Sleepful** [reply]
Thanks! And I just noticed that a transaction may have multiple tags but only one category. So I guess categories enforce that the same transaction won't be counted twice by accident. IDK!
